packages/wonder-blocks-button/components/button-core.js

```diff
@@ -84,8 +85,10 @@ export default class ButtonCore extends React.Component<Props> {
 
         if (!disabled && href) {
             if (clientNav) {
+                // $FlowFixMe
```

Comment:
Huh, now these get caught in `button-core` as well. I spent a short while trying to fix it in `action-item` but wasn't able to.